Draft of the Basic Structure of Government

Defiant-Sheep opened the pull request:
I have written a framework for the structure of the federal government and elections. I had clearer ideas on these topics than on the formulation of rights, and I believe that being clear who makes the decisions will make the rights that must be defended against these decisions more obvious. 
The structure I came up with:
- A bicameral legislature with a powerful upper house reminiscent of America's and Australia's Senate
- An executive branch dependent on the approval of the president and the lower house, similar to France's cabinet in theory
- An independent body, chosen by sortition, that can remove corrupt presidents

I believe that coming up with ideas for the design of institutions that should be insulated from politics (major courts, Central Bank, the military to a degree) should be a separate discussion. 

If my framework is not an acceptable starting point, I hope that it at least prompts discussion.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

The basic structure of government

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defined the election of the cabinet

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Specified the basic legislative procedure.

Defiant-Sheep committed:
Senate Nominations and Election

I believe that this method of nominating candidates to the senate will help to avoid the cronyism that many upper houses suffer from

Defiant-Sheep committed:
Timing of elections

Defiant-Sheep committed:
European Parliament Elections

Made a more sensical system than the one I had before

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

I read the wiki properly today, and finally realised that you wanted the executive branch to have significant independence from the legislature. I am rewriting all of this into a semi-presidential system.

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Legislative Process

Defiant-Sheep committed:
Removed misplaced description of legislative process

Defiant-Sheep committed:
Merge pull request #1 from Defiant-Sheep/Semi-Presidential

I have realised that the main branch serves no clear purpose

Defiant-Sheep committed:
Presidential Elections

I am suggesting that nominations be done by member states to increase the separation of powers. Nomination by petition would lead to hundreds of candidates running, while nomination by the Continental Congress would tie them too closely with their congressional allies. I think member states should nominate multiple candidates to increase diversity of choice.

The first round must happen to allow voters to know who the frontrunners are and to learn about them. The first two parts of the process produce candidates that are trusted outside of their member state, so that it cannot be skewed by the large states. The second two rounds ensure that there are genuine opposition candidates running in case no candidate is approved of by a majority. 

The second round is a Condorcet system. This means that if one candidate is preferred to every other candidate, they will win, even if very few people saw them as the best option. This is less certain in Instant Runoff Voting.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TABLE_OF_CONTENTS.md

Defiant-Sheep committed:
Impeachment

I am adding an institution who can remove the president. We can see in from Trump's second impeachment that the legislature will contain too many loyalists to the president when there is a valid reason to impeach them.

Defiant-Sheep committed:
Practicality concerns

Defiant-Sheep committed:
Created a process to impeach the president

Defiant-Sheep committed:
I figured out how to make a proper list

Defiant-Sheep committed:
Ensured that the Federal Jury contains no lifetime members

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

Defiant-Sheep committed:
Update TABLE_OF_CONTENTS.md

Defiant-Sheep committed:
Presidential succession and no confidence required for vice presidents

I forgot to add a system for the absence of a president earlier. I also realised that requiring parliamentary approval for their appointment would damage separation of powers in this case.

Defiant-Sheep committed:
Missed stuff in last commit

Defiant-Sheep commented:
@Mathew-Iron, are there any particular articles in my fork that you feel are too vague, contrary to the principles you wanted to enshrine, or making major decisions that should be decided collectively?

Mathew-Iron commented:
@Defiant-Sheep Good work but I think we should specify, as in the US constitution, I can see that you've taken a lot of inspiration from it, but the articles about the president are too vague, how government works in more detail, how people are elected, what the different chambers are for, and so on.

I also think we should change the name to “continental congress”, because we want Europe to be unified as a country, with stronger boundaries than different countries on a continent. Something like The Solarium, The Concordium , The Aureum, The Consilium i don't know

That's a great work, thank you so much for your efforts.

https://www.senate.gov/about/origins-foundations/senate-and-constitution/constitution.htm

Defiant-Sheep commented:
@Mathew-Iron Thank you for identifying the general issues; I will try to make my fork more detailed over the week. When you say I was too vague about "how people are elected", do you just mean not describing STV, or do you also mean the presidential elections and the selection of party lists, or do you mean the process of voting and voter identification? If selecting party lists is a major part of the issue, I have to tell you that I have no idea how it should happen.

I will leave choosing a more unique name for the legislature until last, but I will remove the "continental" adjective immediately as it appears to be the main thing bothering you. Do any of the names you listed have distinctive connotations? 

You're welcome, and thank you for the informative comments.

Defiant-Sheep committed:
Changed "Continental Congress" to simply "Congress"

Defiant-Sheep committed:
Appointment of ambassadors and ratification of treaties

Staphylococcus commented:
@Defiant-Sheep Thanks so much for the contribution, one idea that popped into my mind is that it might make sense to limit the scope of the fork. The more that's happening inside one pull request, the more discussion we need to go through to clarify every aspect. We could e.g. merge the general structure first, then refine individual articles.

Defiant-Sheep commented:
> @Defiant-Sheep Thanks so much for the contribution, one idea that popped into my mind is that it might make sense to limit the scope of the fork. The more that's happening inside one pull request, the more discussion we need to go through to clarify every aspect. We could e.g. merge the general structure first, then refine individual articles.

Understood. The general structure is the simple part, so I should be able to open a more limited pull request this evening. Do the two of you want this pull request to remain open for discussion, or am I meant to be the one to close it?

Defiant-Sheep committed:
Mentioned the lower house's powers over the cabinet in the structure of government

Defiant-Sheep committed:
Renamed "Congress" to the "Consilium"

The name was suggested by Mathew Irone. I like it the best out of all the names either of us have thought of.

Defiant-Sheep committed:
Renamed "Congress" to the "Consilium"

Defiant-Sheep committed:
Moved the Structure of Government to its own branch

Staphylococcus asked me to separate this so that the pull requests could be done in manageable portions.

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_1.md

Defiant-Sheep committed:
Update TITLE_2_CH_2.md

After looking at this again, I realised that "Enabling Acts" is a poor term for these kinds of laws.

Defiant-Sheep committed:
Constituency seat allocations

I have no recollection of why I decided to increate the population required for a country to gain more seats in the European Parliament. This is fixed now.

Defiant-Sheep committed:
Defined the Single Transferrable Vote

Defiant-Sheep committed:
Description of referendum

I think that referendums should require a minimum threshold of voter participation to ensure that the people actually wanted them to pass.

Mathew-Iron commented on TITLE_4_CH_3.md:
very good, nothing to complain about, simple and efficient

Defiant-Sheep committed:
Moved president article

I was informed that the president should be described before the cabinet

Defiant-Sheep committed:
Definition for Sortition

I codified a simple definition for sortition.

I cannot think of any reason I might of had for giving all but 2 member states the same number of Federal Jurors, so I changed the math.

Defiant-Sheep committed:
Merge branch 'main' into General-Structure-of-Government

Defiant-Sheep committed:
Merge pull request #2 from Defiant-Sheep/General-Structure-of-Government

General structure of government

Defiant-Sheep committed:
Revert "General structure of government"

Defiant-Sheep committed:
Merge pull request #3 from Defiant-Sheep/revert-2-General-Structure-of-Government

Revert "General structure of government"

Defiant-Sheep committed:
Reduced number of seats in lower house

Defiant-Sheep committed:
Comittees

Defiant-Sheep committed:
Update TITLE_4_CH_1.md

Defiant-Sheep committed:
Clarified term limits if a president is incapable and is replaced

Defiant-Sheep committed:
Made the first part of the first round of the presidential election easier to read

Defiant-Sheep committed:
Update TITLE_4_CH_1.md

Defiant-Sheep committed:
Fixed the equation for reducing voting power of satisfied ballots

I only just noticed that the old equation would effectively forbid satisfied voters from influencing later counts

Defiant-Sheep committed:
Allowing equal rankings in STV
